Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/CheckInWarningMatcher.kt

```diff
@@ -11,17 +11,25 @@ import javax.inject.Inject
 class CheckInWarningMatcher @Inject constructor(
     private val checkInsRepository: CheckInRepository,
     private val traceTimeIntervalWarningRepository: TraceTimeIntervalWarningRepository,
+    private val presenceTracingRiskRepository: PresenceTracingRiskRepository,
     private val dispatcherProvider: DispatcherProvider
 ) {
-    suspend fun execute(): List<CheckInWarningOverlap> {
-        val checkIns = checkInsRepository.allCheckIns.firstOrNull() ?: return emptyList()
+    suspend fun execute() {
```

Comment:
I only see this being called from a test, shouldn't this be called from somewhere?

Reply:
Which PR will add code that calls this?